### PR TITLE
Add approval by default to privacy notices

### PR DESCRIPTION
**Pull request by edwardUL99**
#### Summary 
Quick outline of new learn api requirements form Ayrton and Jacob:
1.	Implement a privacy notice which is “enabled” by default for the learn integration.
2.	Implement an experience for when the privacy notice is declined by either an admin or individual user. This involves showing a UI component on the help pane notifying the user that the experience may be limited.
a.	If admin, the text will have a link for them to open the privacy notices page to enable it
b.	If user and the notice is not yet set by them or an admin, the text will have a link to open the wizard for approval
c.	If user and the notice is disabled by admin, the card will simply tell them to reach out to an administrator
For on-prem, this isn’t supported, so no privacy notice or search box necessary.

To implement 2, the current Privacy notice table does not allow us to differentiate if it is enabled by default. It simply calculates if it is approved/disabled for all users or the user decides. So I am adding a new column which will identify if it is enabled by default so that the checks for approval will return approved if approved by default or admin approved. Hence, the questions on trying to add a new column to the system tables.

#### Work Item(s) 
Fixes [AB#556896](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/556896)

### CODE REVIEW

**Comment by edwardUL99** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by edwardUL99** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: src/System Application/App/Privacy Notice/src/PrivacyNoticeImpl.Codeunit.al [hunk 1]
```diff
@@ -171,13 +184,18 @@ codeunit 1565 "Privacy Notice Impl."
     procedure SetApprovalState(PrivacyNoticeId: Code[50]; PrivacyNoticeApprovalState: Enum "Privacy Notice Approval State")
     var
         PrivacyNoticeApproval: Codeunit "Privacy Notice Approval";
+        PrivacyNotice: Codeunit "Privacy Notice";
```
Comment (haoranpb) [inline src/System Application/App/Privacy Notice/src/PrivacyNoticeImpl.Codeunit.al]: We usuaslly don't reference the facade codeunit from the impl codeunit

Comment (darjoo) [reply src/System Application/App/Privacy Notice/src/PrivacyNoticeImpl.Codeunit.al]: +1

**Comment by darjoo** [comment]
No other comments to add. LGTM besides Haoran's comment.

**Review by haoranpb — APPROVED** [review]
Please make sure to test thoroughly in TIE after merging

**Pull request merged**